lib/data_classes/movie.dart

```diff
@@ -1,46 +1,93 @@
 class Movie {
+  final bool adult;
   final String backdrop;
-  final List<String> genres;
+  final List<int> genres;
+  final int id;
+  final String originalLanguage;
   final String originalTitle;
   final String overview;
+  final double popularity;
   final String poster;
   final DateTime releaseDate;
   final String title;
+  final bool video;
   final double voteAverage;
+  final int voteCount;
   int likes = 0;
 
-  Movie(
-    this.title,
-    this.originalTitle,
-    this.genres,
-    this.poster,
-    this.backdrop,
-    this.releaseDate,
-    this.voteAverage,
-    this.overview,
-  );
+  Movie({
+    required this.adult,
+    required this.backdrop,
+    required this.genres,
+    required this.id,
+    required this.originalLanguage,
+    required this.originalTitle,
+    required this.overview,
+    required this.popularity,
+    required this.poster,
+    required this.releaseDate,
+    required this.title,
+    required this.video,
+    required this.voteAverage,
+    required this.voteCount,
+  });
+
+  factory Movie.fromJson(Map<String, dynamic> j) => Movie(
```

Comment:
try to use a more descriptive name to the parameters that you use, here you can change it to json instead to j

Reply:
Solved